allennlp/tests/training/learning_rate_schedulers_test.py

```diff
@@ -257,8 +257,8 @@ def test_from_params(self):
         assert len(optim.param_groups) == 3
         # The default parameter group in the Optimizer is empty
         assert not optim.param_groups[-1]["params"]
-        assert optim.param_groups[-2]["lr"] == 1.0
-        assert optim.param_groups[-3]["lr"] == 0.5
+        assert optim.param_groups[-2]["lr"] == 1.0 / 32
```

Comment:
I had to search to figure out where this 32 came from - might be nicer to either use `sched.ratio` here, or to do `assert sched.ratio == 32` above, so you just have to look up a few lines to know that 32 is probably sched.ratio.  Or you could just leave a comment saying what this is.